src/pages/creer-un-jeu/index.tsx

```diff
@@ -38,43 +18,33 @@ const activities = [
     label: 'Jeu de la monnaie',
     href: '/creer-un-jeu/monnaie',
     icon: MonnaieIcon,
-    disabled: false,
-    disabledText: '',
+    disabled: true,
+    disabledText: 'Bientôt disponible',
   },
 ];
 
-const Jeu: React.FC = () => {
+const Jeu = () => {
   const router = useRouter();
 
-  const [activitiesWithLinks, suggestionsWithLinks] = React.useMemo(() => {
+  const [activitiesWithLinks] = React.useMemo(() => {
     if ('responseActivityId' in router.query && 'responseActivityType' in router.query) {
       return [
         activities.map((a) => ({
           ...a,
           href: `${a.href}?responseActivityId=${router.query.responseActivityId}&responseActivityType=${router.query.responseActivityType}`,
         })),
-        suggestions.map((s) => ({
-          ...s,
-          href: `${s.href}?responseActivityId=${router.query.responseActivityId}&responseActivityType=${router.query.responseActivityType}`,
-        })),
       ];
     }
-    return [activities, suggestions];
+    return [activities];
```

Comment:
We don't need that anymore, we will redesign later how response activities are handled. But for now you can remove this `activitiesWithLinks` variable and use directly `activities` for the ActivityChoice component.